# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: editor/libeditor/WSRunObject.h
# Commit: 1466d7d61102
# Full Hash: 1466d7d6110279cf167ab8b1567be693df5d7c3c
# Author: Masayuki Nakano <masayuki@d-toybox.com>
# Date: 2020-03-03 09:50:30
# Regressor Bug: 1618089
# File Overlap Count: 2
# Description:
#   Bug 1618089 - part 7: Make some related methods use `EditorDOMPointInText` for arguments or result r=m_kato
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D64338
# ==============================================================================

diff -r 079f4b05ead2 -r 1466d7d61102 editor/libeditor/WSRunObject.h
--- a/editor/libeditor/WSRunObject.h	Tue Mar 03 02:19:59 2020 +0000
+++ b/editor/libeditor/WSRunObject.h	Tue Mar 03 01:15:38 2020 +0000
@@ -812,7 +812,7 @@
    * @return                Start and end of the expanded range.
    */
   template <typename PT, typename CT>
-  Tuple<EditorDOMPoint, EditorDOMPoint> GetASCIIWhitespacesBounds(
+  Tuple<EditorDOMPointInText, EditorDOMPointInText> GetASCIIWhitespacesBounds(
       int16_t aDir, const EditorDOMPointBase<PT, CT>& aPoint) const;
 
   MOZ_CAN_RUN_SCRIPT nsresult CheckTrailingNBSPOfRun(WSFragment* aRun);
